apps/web/components/dashboard/mobile-sidebar.tsx

```diff
@@ -0,0 +1,92 @@
+"use client";
+import { Button } from "@/components/ui/button";
+import { BarChart, BookOpen, FileJson, Settings } from "lucide-react";
+import Link from "next/link";
+import { WorkspaceSwitcher } from "./workspace-switcher";
+import { Workspace } from "@unkey/db";
+import { useEffect, useState } from "react";
+import { AnimatePresence, motion } from "framer-motion";
+import { usePathname } from "next/navigation";
+
+type Props = {
+  workspace: Workspace;
+};
+
+export const MobileSideBar = ({ workspace }: Props) => {
+  const pathname = usePathname();
+  const [hideNav, setHideNav] = useState(true);
+  useEffect(() => {
+    setHideNav(true);
+  }, [pathname]);
+  return (
+    <div className=" ">
+      <div className=" md:hidden p-4 flex items-center justify-between dark:bg-zinc-900 bg-zinc-100">
+        {/* rome-ignore lint/a11y/useKeyWithClickEvents: <explanation> */}
+        <svg
```

Comment:
https://lucide.dev/icons/panel-right-open
how about this instead?
the current one doesn't fit the style I think

Reply:
I actually tried this one and didn't feel it fits either but if u say so sure :)

Reply:
<img width="369" alt="Screenshot 2023-07-21 at 9 47 31 PM" src="https://github.com/unkeyed/unkey/assets/86073083/d7911ccd-ed0c-4aec-8c48-3aa7a5f05774">